app/controllers/api/v0/forms_controller.rb

```diff
@@ -8,17 +8,25 @@ def index
   end
 
   def show
-    @form = current_user.forms.find_by_short_uuid(params[:id])
-    @page_num = (params[:page].present? ? params[:page].to_i : 0)
-    @page_size = (params[:page_size].present? ? params[:page_size].to_i : 500)
-    @page_size = 5000 if @page_size > 5000
+    form = current_user.forms.find_by_short_uuid(params[:id])
+    page_num = (params[:page].present? ? params[:page].to_i : 0)
+    page_size = (params[:page_size].present? ? params[:page_size].to_i : 500)
+    page_size = 5000 if page_size > 5000
+    # Date filter defaults to 1 year ago and 1 day from now
+    # Is there ever a case where we'd want to see submissions older than a year via the API?
```

Comment:
Yes, a Form Manager should be able to see Submissions order than a year, if they specify.
I think a one year ago default works.